index.html

```diff
@@ -0,0 +1,139 @@
+                            <p class = "video1Text">Youtube User
+                            <br>
+                            5.13M subscribers
+                            <br>
+                            2323332 views
+                        </p>
+                    </div>
+                </li>
+                <li>
+                    <div class = "video1">
+                        <a href = "https://www.youtube.com/watch?v=27C-TuJceeI"><img src = "https://i.ytimg.com/vi/27C-TuJceeI/hqdefault.jpg?sqp=-oaymwEZCPYBEIoBSFXyq4qpAwsIARUAAIhCGAFwAQ==&rs=AOn4CLAYiGVtsQ-5EnWeINsqrY6fNl86fg" alt = "Why I quit SE"></a>
+                        <p class = "video1Text">Youtube User
+                            <br>
+                            5.13M subscribers
+                            <br>
+                            2323332 views
+                        </p>
+                    </div>
+                </li>
+                <li>
+                    <div class = "video1">
+                        <a href = "https://www.youtube.com/watch?v=YJZCUhxNCv8"><img src = "https://i.ytimg.com/vi/6790FVXWBw8/hqdefault.jpg?sqp=-oaymwEZCPYBEIoBSFXyq4qpAwsIARUAAIhCGAFwAQ==&rs=AOn4CLB6VyimIMcL1ejfY3Y4DxPvjPT9vA" alt = "how to get a job at 4 big companies?"></a>
+                        <p class = "video1Text">Youtube User
+                            <br>
+                            5.13M subscribers
+                            <br>
+                            2323332 views
+                        </p>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class = "video">
+            <iframe width = "700" height = "400" src = "https://www.youtube.com/embed/rSOzN0eihsE" allow = "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <div class = "videoInfo">
+                <div class = "videoTittle">Akon - Beautiful ft. Colby O'Donis, Kardinal Offishall</div>
+                <div class = "videoIcons">
+                    <p>260.878.442 visualizaciones•16 jun. 2009</p>
+                    <img src = "images/like-icon.PNG" alt = "like logo" id = "likeIcon" width = "23" height = "18">
+                    <p>937.957</p>
+                    <img src = "images/dislike-icon.PNG" alt = "dislike logo" width = "23" height = "18">
+                    <p>31.713</p>
+                    <img src = "images/share-icon.PNG" alt = "share logo" width = "23" height = "18">
+                    <p>Share</p>
+                    <img src = "images/save-icon.PNG" alt = "save logo" width="23" height = "18">
+                    <p>Save</p>
+                </div>
+                <hr>
```

Comment:
Remove these as well and let's use CSS for that, please.

Reply:
The above update applies to these earlier requested changes as well. 👆 Thank you, @adaorachi for the heads up. I owe you one 👍